README.rst

```diff
@@ -1,39 +1,41 @@
-aldryn-style
+============
+Aldryn Style
 ============
 
-A Plugin for django CMS to add CSS classes to other plugins
-
+A Plugin for django CMS to add CSS classes to other plugins.
 
+------------
 Installation
 ------------
 
 This plugin requires `django CMS` 2.4 or higher to be properly installed.
 
-* In your projects `virtualenv`_, run ``pip install aldryn-style``.
+* Create a `virtualenv` and run ``pip install aldryn-style``
 * Add ``'aldryn_style'`` to your ``INSTALLED_APPS`` setting.
 * Run ``manage.py migrate aldryn_style``.
 
-
+-----
 Usage
 -----
 
 You can define styles in your settings.py:
 
 ```
+gettext = lambda s: s
 ALDRYN_STYLE_CLASS_NAMES = (
-    ('info', _("info")),
-    ('new', _("new")),
-    ('hint', _("hint")),
+    ('info', gettext('info')),
+    ('new', gettext('new')),
+    ('hint', gettext('hint')),
 )
```

Comment:
Its a bit more than a just a convention. The GetText tools parse the source files and look for the gettext functions by name OR the "_". The only reason we have to do anything different in the settings.py file is that we don't (want to) import modules there, so doing `from django.utils.translations import ugettext as _` is out of the question. Using the lambda makes it pass Python parsers, and the GetText parser will then still pick it up.

I actually agree with the lambda definition here, @chive, as it makes it clear what the intent is.